railties/lib/rails/code_statistics.rb

```diff
@@ -43,12 +43,18 @@ class CodeStatistics
     HEADERS = { lines: " Lines", code_lines: "   LOC", classes: "Classes", methods: "Methods" }
 
     class_attribute :directories, default: DIRECTORIES
+    class_attribute :test_types, default: TEST_TYPES
 
     # Add directories to the output of the `bin/rails stats` command.
     #
     #   Rails::CodeStatistics.register_directory("My Directory", "path/to/dir")
-    def self.register_directory(label, path)
+    #
+    # For directories that contain test code, set the `test_directory` argument to true.
+    #
+    #   Rails::CodeStatistics.register_directory("Model specs", "spec/models", test_directory: true)
+    def self.register_directory(label, path, test_directory: false)
```

Comment:
Looks like the `test_directory` is unused in the code.

Reply:
Fixed

Reply:
Argh 🙃 .
Thanks @kamipo @rafaelfranca !